src/include/partitioning/custom_part_lib.rb

```diff
@@ -293,13 +293,17 @@ def check_ok_fssize(size_k, volume)
 
       if Ops.less_than(size_k, min_size_k)
         # warning message, %1 is replaced by fs name (e.g. Ext3)
-        # %2 is prelaced by a size (e.g. 10 MB)
+        # %2 is replaced by the user inputted size (e.g. 20GB)
+        # %3 is replaced by a size (e.g. 10 MB)
         Popup.Warning(
           Builtins.sformat(
             _(
-              "Your partition is too small to use %1.\nThe minimum size for this file system is %2.\n"
+              "Your partition is too small to use %1.\n" +
+                "Your inputted size (after rounding up) is %2.\n" +
```

Comment:
This is not proper English. Please change this to

"The size you entered (after rounding up) is %2.\n"